### DISCUSSION TITLE
BIRT Eclipse 4.6.3 - when exporting to Word, I get Gibberish if there's any table groups

### DISCUSSION THREAD

**Discussion by stevehydro** [eclipse-birt/birt#1523] [Q&A] [answered]
As the title says, whenever I export to Word, it creates gibberish if there's a Group within a table, even when it's blank.  No formatting was done to the table, just inserted it and added a group.

Exporting to anything else works just fine.

any thoughts?

![Capture](https://github.com/eclipse-birt/birt/assets/142328454/440af289-bb0c-4130-9a3c-ae52d2006592)

**Comment by stevehydro** [comment]
![Capture2](https://github.com/eclipse-birt/birt/assets/142328454/42d05c99-9f0c-478e-bf3f-5c5210b37630)

**Comment by hvbtup** [comment]
That's not gibberish, that's just a text mark. It seems you have enabled showing field code in Word.

**Reply by stevehydro** [reply]
unfortunately that was already disabled, also, it only happens when I add a Group to a table.  As soon as I remove the Group, it disappears.

**Reply by hvbtup** [reply]
Verify your settings in Word. It is a text mark, or to be more precise, the target for a TOC entry.
If you create a group to a table or list item, then BIRT will automatically creata a TOC entry for it. If you don't want this, you can just clear the corresponding field in the group property dialog.

**Reply by stevehydro** [reply]
which field is making the TOC entry? 
![Capture](https://github.com/eclipse-birt/birt/assets/142328454/8db6b49f-6488-4a4d-bc68-615bc6ef717e)

**Reply by speckyspooky** [reply, marked as answer]
Remove the value 'row["BCHSOLDDA"]' of "Item Expression" of the frame "Table of Content" (= TOC) and try it again.

**Reply by stevehydro** [reply]
That was it!

Thank you so much
